litestar/template/base.py

```diff
@@ -55,10 +65,10 @@ def csrf_token(context: TemplateContext) -> str:
     Returns:
         A CSRF token if the app level ``csrf_config`` is set, otherwise an empty string.
     """
-    return context["request"].scope.get("_csrf_token", "")  # type: ignore
+    return _get_request_from_context(context).scope.get("_csrf_token", "")  # type: ignore[return-value]
```

Comment:
`object` return here, because Request.scope is an instance of the typed-dict HTTPScope which doesn't have a `_csrf_token` key on it. I think `_csrf_token` would be better off stored/retrieved from scope state using `utils.scope.{get_litestar_scope_state,set_litestar_scope_state}`.